rs/nns/governance/src/network_economics.rs

```diff
@@ -351,6 +360,21 @@ impl VotingPowerEconomics {
             defects.push("clear_following_after_seconds must be set.".to_string());
         }
 
+        if let Some(delay) = self.neuron_minimum_dissolve_delay_to_vote_seconds {
+            pub const MIN_DISSOLVE_DELAY_FOR_VOTE_ELIGIBILITY_SECONDS_BOUNDS: RangeInclusive<u64> =
+                (3 * ONE_MONTH_SECONDS)..=(6 * ONE_MONTH_SECONDS);
+
+            if !MIN_DISSOLVE_DELAY_FOR_VOTE_ELIGIBILITY_SECONDS_BOUNDS.contains(&delay) {
```

Comment:
I'm confused by this - why not just do >= and <= instead of constructing a range and seeing if the value is inside of it?